pilot/pkg/networking/core/v1alpha3/tracing.go

```diff
@@ -138,6 +138,14 @@ func configureFromProviderConfig(pushCtx *model.PushContext, meta *model.NodeMet
 	case *meshconfig.MeshConfig_ExtensionProvider_Skywalking:
 		tracing, err = buildHCMTracing(pushCtx, providerCfg.Name, provider.Skywalking.Service,
 			provider.Skywalking.Port, 0, func(clusterName string) (*anypb.Any, error) {
+				clientConfig := &tracingcfg.ClientConfig{}
+
+				if provider.Skywalking.AccessToken != "" {
+					clientConfig.BackendTokenSpecifier = &tracingcfg.ClientConfig_BackendToken{
+						BackendToken: provider.Skywalking.AccessToken,
```

Comment:
Is this value obfuscated in /config_dump?

I guess istiod debug interface will also show it.... holding access tokens in mesh config seems like a bad idea.

Reply:
Agree, it has security issue to expose token in configmap